airflow/kubernetes/pod_generator.py

```diff
@@ -237,7 +243,8 @@ def __init__(  # pylint: disable=too-many-arguments,too-many-locals
 
         # Pod Container
         self.container = k8s.V1Container(name='base')
-        self.container.image = image
+        self.container.image = self.qualify_image_url(image)
```

Comment:
Please revert this line, because self.container.image can be a full qualified string with private registry in the beginning.

Reply:
Yes you're right. I don't want to change the default behavior for the default image. I just want to fix the side-car hard coded  image path.